src/Elcodi/Fixtures/DataFixtures/ORM/Base/Page/PageData.php

```diff
@@ -15,7 +15,7 @@
  * @author Elcodi Team <tech@elcodi.com>
  */
 
-namespace Elcodi\Fixtures\DataFixtures\ORM\Page;
+namespace Elcodi\Fixtures\DataFixtures\ORM\Base\Page;
```

Comment:
I'm not sure that pageData shoud be part of the "base" fixtures. If you want an empty store you don't want any post, what do you think?